### PR TITLE
chore: make it possible to filter internal IP address

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

**Review by dolcalmi — COMMENTED** [review]
looks good, just a question...dont we want to use something more generic? like https://stackoverflow.com/a/13969691 or use a library like `ipaddr.js` https://github.com/whitequark/ipaddr.js/blob/master/test/ipaddr.test.js#L150

**Comment by nicolasburtey** [comment]
> looks good, just a question...dont we want to use something more generic? like https://stackoverflow.com/a/13969691 or use a library like `ipaddr.js` https://github.com/whitequark/ipaddr.js/blob/master/test/ipaddr.test.js#L150

yeah that would remove the additional configuration parameter I have added. that is a good point. I'll do that.

**Comment by nicolasburtey** [comment]
ok I think that is better. @dolcalmi let me know

File: test/unit/services/ipfetcher/fetch-ip-info.spec.ts [hunk 1]
```diff
@@ -59,6 +60,28 @@ describe("IpFetcher - fetchIPInfo", () => {
   })
 })
 
+describe("ipaddr.js", () => {
```
Comment (dolcalmi) [inline test/unit/services/ipfetcher/fetch-ip-info.spec.ts:63]: this needs to be moved to test domain folder and description should be isPrivateIp (we are testing domain function here)

**Review by dolcalmi — CHANGES_REQUESTED** [review]
looks good, just a minor change

**Pull request merged**